<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sparkWidget</class>
 <widget class="QWidget" name="sparkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>spark</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="GLWidget" name="OpenGL" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtOpenGL/glwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
